src/robusta/runner/web.py

```diff
@@ -57,11 +69,40 @@ def handle_alert_event():
         alert_manager_event = AlertManagerEvent(**req_json)
         for alert in alert_manager_event.alerts:
             alert = Web._relabel_alert(alert)
-            Web.alerts_queue.add_task(Web.event_handler.handle_trigger, PrometheusTriggerEvent(alert=alert))
+            alert_hash = Web.get_compound_hash([
+                alert.fingerprint.encode('ascii'),
+                alert.status.encode('utf-8'),
+                str(alert.startsAt.timestamp()).encode('ascii'),
```

Comment:
please also add the `endsAt`
I don't think you can get 2 alerts with everything the same, and only endsAt is different, but I don't see a downside in adding it

Reply:
Added